packages/web/hooks/use-has-asset-variants.tsx

```diff
@@ -54,7 +54,7 @@ export const useHasAssetVariants = () => {
         // note - there is some local storage order issues when navigating between pages, so hasSeenToastThisSession is a failsafe
         if (doNotShowAgain === true || hasSeenToastThisSession) return;
 
-        const hasAssetsToConvert = data?.hasVariants ?? false;
+        const hasAssetsToConvert = data?.assetVariants?.length > 0 || false;
```

Comment:
Is `|| false` needed?

From my own console, it appears `undefined > 0` is `false`, which is good